docs/components/password/index.html

```diff
@@ -0,0 +1,143 @@
+---
+title: Password
+minver: 0.19.0
+also:
+  components/password/test.html: Testing - Password
+  elements/hx-password-control/index.html: <hx-password-control>
+---
+{% extends 'component.njk' %}
+
+{% block page_header %}
+  <p>
+    Use {{page.title}} anytime a user must enter or update a password.
+  </p>
+  <p>
+    NOTE: This document is the BETA version of this input.
+  </p>
+{% endblock %}
+
+{% block content %}
+  <section>
+    <header>
+      <h2 id="basic-password-input">Password</h2>
+      {# TODO: add section description #}
+    </header>
+     <p>Build out the password entry interaction including:
+        Meeting or not meeting password requirements.  Consider a popover on focus that interactively displays met criteria.
+        Showing and hiding password text.
+    </p>
+
+    <div class="example" id="vue-passwordInputDemo" v-cloak>
+      <header>
+        <form class="beta-hxForm">
+          <hx-password-control>
+            <input
+              id="pwdLabel"
+              type="password"
+              v-model="label"
+              minlength="8"
+              placeholder="Please Enter Password"
+              pattern="(?=.*\d)(?=.*[a-z])(?=.*[A-Z]).{8,}"
+              required
+            />
+            <label for="pwdLabel">
```

Comment:
### The following sections should be removed from this demo:

<img width="808" alt="Screen Shot 2020-02-26 at 1 04 48 PM" src="https://user-images.githubusercontent.com/10120600/75380385-8d7ad000-589c-11ea-97f5-0379f981f020.png">

Reply:
Changes have been done.